CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing
+
+Contributions are welcome from anyone.
+
+To report bugs or issues, open [new issues](https://github.com/brndnmtthws/conky/issues/new).
+To submit code changes, open [new pull requests](https://github.com/brndnmtthws/conky/compare).
+
+## Guidelines
+
+Patches submitted in issues, email, or elsewhere may be ignored. When submitting PRs, please:
+
+- Describe the changes, why they were necessary, etc
+- Describe how the changes affect existing behaviour
+- Describe how you tested and validated your changes
+- Include unit tests when appropriate
+- Include any relevant screenshots/evidence demonstrating that the changes work and have been tested
+- Any new source files should include a GPLv3 license header
+- Any contributed code must be GPLv3 licensed
+- Always leave the code better than you found it
```

Comment:
This feels like an impossible thing to do. :) Maybe don't add that one.

Reply:
I'll change "always" to "try".

Reply:
"Leave the code better than you found it" sounds great too.

Reply:
I'll stick with the "try" wording, because I don't want this to be a hard requirement.

I do, however, want to make it clear that I may reject PRs simply for style violations.